### DISCUSSION TITLE
All class signatures, function signatures and method signatures in the documentation are shown incorrectly

### DISCUSSION THREAD

**Discussion by imthematrix** [fastapi/fastapi#11276] [Questions]
### Description

All class signatures, function signatures and method signatures in FastAPI's documentation are rendered incorrectly. For example, the class signature of the `fastapi.FastAPI` class looks like this:

`FastAPI(*, <span data-autorefs-optional="fastapi.FastAPI(debug)">debug</span>=False, <span data-autorefs-optional="fastapi.FastAPI(routes)">routes</span>=None, <span data-autorefs-optional="fastapi.FastAPI(title)">title</span>='FastAPI', `...`)`

You can see the `<span>` tags on the website [FastAPI class - FastAPI](https://fastapi.tiangolo.com/reference/fastapi/).

This applies to all code blocks in the documentation that have the CSS classes `doc-signature` and `highlight` (`class="doc-signature highlight"`). I didn't notice any code blocks like that in the Turorial [Learn - FastAPI](https://fastapi.tiangolo.com/learn/) but there are many in the API reference [Reference - Code API - FastAPI](https://fastapi.tiangolo.com/reference/).

<sub>Related to discussion #11257</sub>

**Comment by YuriiMotov** [comment]
Fixed
